File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task133_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <string>

int main() {
    std::string username, oldPassword, newPassword, confirmPassword;

    std::cout << "Enter your username: ";
    std::getline(std::cin, username);
    std::cout << "Enter your old password: ";
    std::getline(std::cin, oldPassword);
    std::cout << "Enter your new password: ";
    std::getline(std::cin, newPassword);
    std::cout << "Confirm your new password: ";
    std::getline(std::cin, confirmPassword);

    if (newPassword == confirmPassword) {
        std::cout << "Password reset successfully!" << std::endl;
    } else {
        std::cout << "New password and confirm password do not match. Please try again." << std::endl;
    }

    return 0;
}

File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task133_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.util.Scanner;

public class PasswordReset {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter your username: ");
        String username = scanner.nextLine();
        System.out.print("Enter your old password: ");
        String oldPassword = scanner.nextLine();
        System.out.print("Enter your new password: ");
        String newPassword = scanner.nextLine();
        System.out.print("Confirm your new password: ");
        String confirmPassword = scanner.nextLine();

        if (newPassword.equals(confirmPassword)) {
            System.out.println("Password reset successfully!");
        } else {
            System.out.println("New password and confirm password do not match. Please try again.");
        }
    }
}